File: vote.cpp
#include<iostream>
using namespace std;
class decisionEx
{
    int n;
    public: void check_age()
    {
        cout<<"Enter the age of person";
        cin>>n;
        if(n>=18)
        {
         cout<<"Eligible for vote";
        }
        else{
            cout<<"Not eligible to vote";

        }
    }

};
int main()
{
    decisionEx d1;
    d1.check_age();

}

File: Assign2/vote.java
package Assign2;
import java.util.Scanner;
public class vote
{
    public static void main(String args[])
    {
        int age;
        System.out.println("Enter the Age : ");
        Scanner sc = new Scanner(System.in);
        age = sc.nextInt();	
if(age>=18)
        {
            System.out.println("The Person is eligible to vote");
        }
      else
        {
            System.out.println("The Person is not eligible to vote");
        }
    }
}
 
